src/lib.rs

```diff
@@ -1285,10 +1297,18 @@ impl From<chain::fragment::FragmentId> for FragmentId {
 
 #[wasm_bindgen]
 impl FragmentId {
-    pub fn from_bytes(bytes: &[u8]) -> FragmentId {
+    pub fn calculate(bytes: &[u8]) -> FragmentId {
         chain::fragment::FragmentId::hash_bytes(bytes).into()
     }
 
+    pub fn from_bytes(bytes: Uint8Array) -> Result<FragmentId, JsValue> {
```

Comment:
This is indeed a better method now, although I'm worried with an unspecified amount of javascript need to be migrated of to this new method